css/style.css

```diff
@@ -6,3 +6,55 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+  body {
+background-color: rgb(227, 237, 241);
+margin: 2em;
+padding: 1rem;
+display: flex;
+flex-direction: column;
+ }
+
+ h1{
+color: black;
+text-align: center;
+}
+nav{
+    text-align: center;
+    margin: 4rem;
+}
+main{
+    margin: 3em;
+    padding: 1em;
+    padding-left: 23em;
```

Comment:
It's rarely wise to use padding to position things. And even more risky to use em sizing. Try zooming your browser to 200% to find out why! 😉 

https://www.boia.org/blog/give-yourself-an-accessibility-test-zoom-your-page-to-200